app/controllers/catalog_controller.rb

```diff
@@ -2,7 +2,6 @@ class CatalogController < ApplicationController
   include Hydra::Catalog
   include Hydra::Controller::ControllerBehavior
   include Sufia::Catalog
-  include BlacklightAdvancedSearch::Controller
 
```

Comment:
Sufia doesn't expose any advanced search in its ui as far as I know. the choices were to remove the advanced search stuff from this controller or add the advanced search gem into our Gemfile. It's good to call this out as something to double-check in testing.